app/backend/src/tests/test_auth.py

```diff
@@ -269,9 +269,25 @@ def test_signup_invalid_birthdate(db):
                 radius=100,
                 accept_tos=True,
             )
-        )
+        )        
     assert e.value.code() == grpc.StatusCode.INVALID_ARGUMENT
     assert e.value.details() == errors.INVALID_BIRTHDATE
 
+    with auth_api_session() as (auth_api, metadata_interceptor):
```

Comment:
you can use the same session as above: you'll need to separate the `pytest.raises` into it's own `with` (just break it at the comma and create a new with block to put the above in, or have a look at how `pytest.raises` is used elsewhere